### PR TITLE
Updating RGW reef suites for executions

### PR DESCRIPTION
**Pull request by anrao19**
# Description
Updating RGW reef suites for executions
https://issues.redhat.com/browse/RHCEPHQE-10499

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/reef/rgw/tier-1_rgw_archive_ms_upgrade_61GA_to_7x_latest.yaml [hunk 1]
```diff
@@ -0,0 +1,423 @@
+#upgrade multisite with archive zone to latest 5.x(GA) to 6x
```
Comment (ckulal) [inline suites/reef/rgw/tier-1_rgw_archive_ms_upgrade_61GA_to_7x_latest.yaml:1]: documentations needs update!

File: suites/reef/rgw/tier-1_rgw_ms_upgrade_5_3GA_to_7x_latest.yaml [hunk 2]
```diff
@@ -0,0 +1,437 @@
+# Test suite for evaluating RGW multi-site deployment scenario.
```
Comment (ckulal) [inline suites/reef/rgw/tier-1_rgw_ms_upgrade_5_3GA_to_7x_latest.yaml:1]: documentation needs update!

**Review by ckulal — COMMENTED** [review]
needed some minor documentation update, remaining changes looks good!

File: suites/reef/rgw/tier-1_rgw_ms_upgrade_5_3GA_to_7x_latest.yaml (hunk 2 above)
Comment (anrao19) [reply suites/reef/rgw/tier-1_rgw_ms_upgrade_5_3GA_to_7x_latest.yaml]: @ckulal done

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**